### PR TITLE
Increase precision for benchmarking results in integrity-checker

### PR DESCRIPTION
**Pull request by tudor-da**
CHANGELOG_BEGIN
CHANGELOG_END

### Pull Request Checklist

- [x] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [x] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [x] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: ledger/participant-state/kvutils/tools/src/main/scala/ledger/participant/state/kvutils/tools/integritycheck/IntegrityChecker.scala [hunk 1]
```diff
@@ -172,8 +172,8 @@ class IntegrityChecker[LogResult](commitStrategySupport: CommitStrategySupport[L
       case Success((startTime, _)) =>
         Success {
           println("Successfully indexed all updates.".green)
-          val durationSeconds = Duration.fromNanos(System.nanoTime() - startTime).toSeconds
-          val updatesPerSecond = readService.updateCount() / durationSeconds.toDouble
+          val durationSeconds = Duration.fromNanos(System.nanoTime() - startTime).toMillis / 1000.0d
```
Comment (SamirTalwar-DA) [inline ledger/participant-state/kvutils/tools/src/main/scala/ledger/participant/state/kvutils/tools/integritycheck/IntegrityChecker.scala:175]: I'd prefer an explicit `toDouble` here.

Also, the `d` suffix isn't necessary; doubles are the default.

```suggestion
          val durationSeconds = Duration.fromNanos(System.nanoTime() - startTime).toMillis.toDouble / 1000.0
```

**Review by SamirTalwar-DA — APPROVED** [review]
Oh, good, this was really annoying me yesterday. Nice catch!

File: ledger/participant-state/kvutils/tools/src/main/scala/ledger/participant/state/kvutils/tools/integritycheck/IntegrityChecker.scala (hunk 1 above)
Comment (SamirTalwar-DA) [reply ledger/participant-state/kvutils/tools/src/main/scala/ledger/participant/state/kvutils/tools/integritycheck/IntegrityChecker.scala]: You need to reformat.

**Review by miklos-da — APPROVED** [review]

**Review by fabiotudone-da — APPROVED** [review]

**Pull request merged**